test/processors/audio_processor_test.rb

```diff
@@ -353,12 +353,15 @@ def travis?
         info: {
           size: 90696,
           content_type: 'audio/mpeg',
+          format: 'mp3',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.